Adyen/UI/Form/FormViewController+ViewProtocol.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+public protocol FormViewProtocol {
```

Comment:
no documentation? can you add `/// :nodoc:`

Reply:
done